tools/picard.wdl

```diff
@@ -819,7 +819,7 @@ task bam_to_fastq {
         prefix: "Prefix for the <type of file> file. The extension `<extension>` will be added."
         paired: {
             description: "Is the data Paired-End (true) or Single-End (false)?",
-            common: true,
+            group: "Common",
```

Comment:
I realize I've skipped past a lot of these, but I think I prefer `common` to `Common`. Not enough to hold up the PR though.

Reply:
I do agree that `common` looks better in code format. But for doc rendering, don't we want the titles for each param table capitalized? I suppose we could handle the capitalization within `wdl-doc` by just assuming that no one wants their table titles to be in lowercase.

Reply:
That was my thought. Take the value of `group` and capitalize the first letter. Maybe that's not the right approach to manipulate the values though.

Reply:
> Maybe that's not the right approach to manipulate the values though.

Can you elaborate here?

Reply:
After I wrote it, I thought maybe we shouldn't manipulate the users' specified values. We'd be making assumptions about how to manipulate those (capitalize).

Reply:
I think we can do it intelligently without too much overhead/complexity. Here's my thinking for how we handle it:
If a `group` value is all lowercase, assume the user wants it to be capitalized in the docs (at least in the title[s]). If there's any kind of funky capitalization, leave it be and use as is.